accel/opencl/ops_opencl.py

```diff
@@ -158,19 +188,20 @@ def image(self):
       assert len(self.shape) == 3 and self.shape[2] == 4, f"bad shape for image {self.shape}"
       self._image = CLImage(shape=(self.shape[1], self.shape[0]))
       if self._buf is not None:
-        assert prod(self.shape) == prod(self._image.cl.shape)*4
+        assert prod(self.shape) <= prod(self._image.cl.shape)*4
```

Comment:
Why is this is <= ?

Reply:
The currently tiling algorithm just splits the longest dimension mod `max_hw`, rather than trying to evenly split the longer dimension into N even chunks of size < `max_hw`. So there's potentially unused space in the underlying image.

<img width="845" alt="image" src="https://user-images.githubusercontent.com/8733615/189791051-9c86942f-9cd4-40e5-a032-c1e5b3123448.png">

Reply:
Hmm, I see